Decred Wallet/Features/Backup/BackupVerifyButton.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  BackupVerifyButton.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+
+class BackupVerifyButton: UIButton {
```

Comment:
The properties added here can be added the custom `Button` class, instead of creating a new subclass.